sabnzbd/skintext.py

```diff
@@ -892,6 +891,7 @@
     'Glitter-confirmClear1Download' : TT('Are you sure?'),
     'Glitter-grabbing' : TT('Grabbing NZB...'),
     'Glitter-updateAvailable' : TT('Update Available!'),
+    'Glitter-orphanedJobsMsg' : TT('There are orphaned job folders, click on ICON_PLACEHOLDER to take care of them'),
```

Comment:
Tricky method, translators may not understand this.
You can add an explanatory text by adding this to the end of the line:
`#: WARNING: Leave ICON_PLACEHOLDER untranslated.`
This will show up in the translation tool.